packages/c/clientworker.json

```
@@ -0,0 +1,40 @@
+{
+    "name": "clientworker",
+    "filename": "dist/cw.js",
+    "description": "A simple, globally hijacked, easily customizable, Service Worker-based front-end worker",
+    "homepage": "https://clientworker.js.org",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/ChenYFan/ClientWorker.git"
+    },
+    "keywords": [
+        "ServiceWorker",
+        "ClientWorker",
+        "Accelerate"
+    ],
+    "license": "GPL-3.0-or-later",
+    "autoupdate": {
+        "source": "npm",
+        "target": "clientworker",
+        "fileMap": [
+            {
+                "basePath": "",
+                "files": [
+                    "**/*.js",
+                    "**/*.css",
+                    "**/*.html",
+                    "**/*.woff2",
+                    "**/*.md",
+                    "**/*.json",
+                    "**/*.yaml"
```

👋 Two thoughts here:

1. We generally try to avoid using globstars (`**/*`) as they can match things we don't want. Please try to be explicit about the files you want to match (for example, `dist/*.js`)

2. cdnjs does not support distributing `.html` files, nor do we support `.md` or `.yaml`.

---

I suspect you just want us to be distributing the JavaScipt within your `dist` directory, so all you need here would be:

```suggestion
                    "dist/*.js"
```

---

> 👋 Two thoughts here:
> 
> 1. We generally try to avoid using globstars (`**/*`) as they can match things we don't want. Please try to be explicit about the files you want to match (for example, `dist/*.js`)
> 2. cdnjs does not support distributing `.html` files, nor do we support `.md` or `.yaml`.
> 
> I suspect you just want us to be distributing the JavaScipt within your `dist` directory, so all you need here would be:

Well, if cdnjs really doesn't support distributing html...clientworker is mainly used for mirroring acceleration URLs, my original purpose was to use cdnjs as an example to show the acceleration effect of clientworker.

As for the accelerated content, yaml is cw's accelerated sample file and hot update content, I think it is necessary to exist.

---

In addition, the original purpose of clientworker was to be used as an import method similar to the esm module, so I didn't want to only accelerate the dist folder

---

cdnjs is not a site host -- cdnjs hosts web libraries that can then be loaded on other websites. Is that how you expect folks to be using your library? Including the JavaScript file on their own sites via a script tag?